src/scss/components/forms/_input-checkbox.scss

```diff
@@ -1,18 +1,18 @@
-@if output(form-checkbox) {
-	input[type="checkbox"] {
+@mixin henris-checkbox($el: '.input-checkbox') {
+	input[type='checkbox'] {
```

Comment:
Kan deze zonder het input element? Dit is al best een specefieke naam, kan problemen veroorzaken wanneer je maar 1 class hebt

Reply:
Ok, goed punt.

Reply:
#fixed